### PR TITLE
Added guzzle Verify SSL option for local development

### PR DESCRIPTION
**Pull request by vitalyiegorov**
Usually for local development you create self signed SSL certificates, this PR resolves guzzle/curl error:

```
cURL error 60: SSL certificate problem: self signed certificate (see http://curl.haxx.se/libcurl/c/libcurl-errors.html) (GuzzleHttp\Exception\RequestException)
```

### CODE REVIEW

**Comment by vitalyiegorov** [comment]
@christeredvartsen Closes #65

**Comment by christeredvartsen** [comment]
Would it be better to just forward whatever is set in the `apiClient` part of the extension configuration to the Guzzle constructor, instead of specifying specific options?

You could then configure the Behat extension like this:

```yml
default:
    extensions:
        Imbo\BehatApiExtension:
            apiClient:
                base_uri: https://some.uri
                verify: false
                headers: 
                    x-some-header: value
```

**Comment by vitalyiegorov** [comment]
@christeredvartsen But then we would not able to verify config with builder:
```
public function configure(ArrayNodeDefinition $builder) 
...
```
What do you think?

**Comment by vitalyiegorov** [comment]
@christeredvartsen Updated PR and tests to pass whole array from 
```
apiClient:
   ...
```
to Guzzle client initialization

**Comment by vitalyiegorov** [comment]
@christeredvartsen Can we merge this PR?

**Comment by christeredvartsen** [comment]
> Can we merge this PR?

Hold your :horse: there cowboy. I'll have a look at the code changes after work.

**Comment by vitalyiegorov** [comment]
@christeredvartsen Howdy Sheriff, have you had a chance to check it?

File: src/ServiceContainer/BehatApiExtension.php [hunk 1]
```diff
@@ -73,6 +73,9 @@ public function configure(ArrayNodeDefinition $builder) {
                 ->arrayNode('apiClient')
                     ->addDefaultsIfNotSet()
                     ->children()
+                        ->scalarNode('verify_ssl')
```
Comment (christeredvartsen) [inline src/ServiceContainer/BehatApiExtension.php]: This is no longer needed if we are to simply pass in the array directly to the client.

File: tests/ServiceContainer/BehatApiExtensionTest.php [hunk 2]
```diff
@@ -95,17 +96,21 @@ public function testCanOverrideDefaultValuesWhenBuildingConfiguration() {
...
 
         $baseUri = sprintf('http://localhost:%d', $port);
+        $verifySSL = false;
         $config = (new Processor())->process($rootNode->getNode(true), [
             'api_extension' => [
                 'apiClient' => [
                     'base_uri' => $baseUri,
+                    'verify_ssl' => $verifySSL,
                 ],
+
```
Comment (christeredvartsen) [inline tests/ServiceContainer/BehatApiExtensionTest.php]: Please remove empty line.

**Comment by christeredvartsen** [comment]
Superseded by #67.

**Pull request closed without merging**